fhir_2_ada-r4/mp/9.3.0/payload/2.0.0-beta.1/mp-MedicationAdministration.xsl

```diff
@@ -28,6 +28,7 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
 			<!-- toedienings_datum_tijd -->
 			<xsl:apply-templates select="f:effectiveDateTime" mode="#current"/>
 			<!-- afgesproken_datum_tijd -->
+			<!-- MP-1408 LR: afgesproken_datum_tijd nolonger supported from MP 9.3 beta.3 onwards but kept in stylesheet due to backwards compatibility-->
```

Comment:
I don't think it's necessary to add comments in the 2.0.0-beta.1 stylesheets, so this one and the one below may be removed.

Reply:
This one hasn't been removed yet ;)